AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -155,7 +151,6 @@ - (void)_initializeInstance
   }
   _methodOverrides = overrides;
 
-  _flexBasis = ASRelativeDimensionUnconstrained;
```

Comment:
Sorry but I don't have a very good memory. Looking at the commit history, I think I added this to make sure _flexBasis always has a correct default value. The same thing might be done in the new ASLayoutOptions class [here](https://github.com/facebook/AsyncDisplayKit/pull/644/files#diff-753574fd0cd2c18cdce87fd5393d7734R100).